pkg/jobs/jobsprotectedts/jobs_protected_ts.go

```diff
@@ -20,61 +20,101 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptpb"
 	"github.com/cockroachdb/cockroach/pkg/kv/kvserver/protectedts/ptreconcile"
 	"github.com/cockroachdb/cockroach/pkg/roachpb"
+	"github.com/cockroachdb/cockroach/pkg/scheduledjobs"
+	"github.com/cockroachdb/cockroach/pkg/sql/sqlutil"
 	"github.com/cockroachdb/cockroach/pkg/util/hlc"
 	"github.com/cockroachdb/cockroach/pkg/util/uuid"
 	"github.com/cockroachdb/errors"
 )
 
-// MetaType is the value used in the ptpb.Record.MetaType field for records
-// associated with jobs.
+// MetaType represents the types of meta values we support for records
+// associated with jobs schedules.
+type MetaType int
+
+const (
+	// Jobs is the meta type for records associated with jobs.
+	Jobs MetaType = iota
+	// Schedules is the meta type for records associated with scheduled jobs.
```

Comment:
Would it be fair to say that the records are associated with the schedules themselves rather than scheduled jobs?